e2e_test/e2e_test.go

```diff
@@ -0,0 +1,38 @@
+package e2e_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/celo-org/celo-blockchain/test"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	// This statement is commented out but left here since its very useful for
+	// debugging problems and its non trivial to construct.
+	//
+	//log.Root().SetHandler(log.LvlFilterHandler(log.LvlTrace, log.StreamHandler(os.Stdout, log.TerminalFormat(true))))
+}
+
+// This test starts a network submits a transaction and waits for the whole
+// network to process the transaction.
+func TestSendCelo(t *testing.T) {
+	accounts := test.Accounts(3)
```

Comment:
nit: this produces an account config, not the accounts.

Reply:
Even though the object type name is AccountsConfig if you look at how that variable is used I think accounts is actually an appropriate name.
```
accounts.ValidatorAccounts()
accounts.AdminAccount()
accounts.DeveloperAccounts()
```

There's actually a nice article that discusses the practice of naming things after their types
https://dave.cheney.net/2019/01/29/you-shouldnt-name-your-variables-after-their-types-for-the-same-reason-you-wouldnt-name-your-pets-dog-or-cat